src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java

```diff
@@ -1,20 +1,22 @@
 package com.wikia.webdriver.testcases.facebooktests;
 
-import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.properties.Credentials;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
-import com.wikia.webdriver.elements.mercury.pages.login.RegisterPage;
-import com.wikia.webdriver.elements.mercury.pages.login.SignInPage;
-import com.wikia.webdriver.pageobjectsfactory.componentobject.AuthModal;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.AuthPageContext;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.auth.AuthPageContext'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)